recipes/rainflow/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
When I look at the tarball from pypi.io, it appears as if the license file is already included in the source distribution. Please verify and delete the manually packaged copy if that's the case.

Reply:
@synapticarbors  True. Duplicate license now deleted. 
I guess I misinterpreted the conda-forge instructions!